stock_dropshipping_dual_invoice/tests/test_need_two_invoices.py

```diff
@@ -73,14 +73,30 @@ def test_so_on_delivery_need_one_invoice(self):
         }).create({})
         self.assertIs(False, wizard._need_two_invoices())
 
+    def test_so_mto_need_one_invoice(self):
+        self.so_mto.order_policy = 'picking'
```

Comment:
You can reuse the same `self.so` here and just do `self.so.route_id = ... `. It will be a new, draft one because every test is in its own transaction.